public/api/conf/1.0/application.raml

```diff
@@ -94,80 +94,102 @@ documentation:
 
-        400:
-          body:
-            application/json:
-              type: types.errorResponse
-              examples:
-                missingHeader:
-                  description: Missing Header
-                  value:
-                    code: MISSING_HEADER
-                invalidHeader:
-                  description: Invalid Header
-                  value:
-                    code: INVALID_HEADER
-                invalidNino:
-                  description: Missing Nino
-                  value:
-                    code: INVALID_NINO
+              The two possible values are ‘DA2_BS_ATTENDED’ or ‘DA2_BS_UNATTENDED’.
+            type: string
+            required: true
+            example: DA2_BS_ATTENDED
+          Staff-Id:
+            description: |
+              The Staff identifier associated with the human attending the operation that generated this request.
 
-        404:
-          body:
-            application/json:
-              type: types.errorResponse
-              examples:
-                resourceNotFound:
-                  description: Resource specified by the supplied NINO was not found
-                  value:
-                    code: RESOURCE_NOT_FOUND
+              Must be a 7 digit numeric value
+            type: string
+            required: true
+            example: "1234567"
+        responses:
+          200:
+            headers:
+              Correlation-Id:
+                description: A unique UUID for this request
+                type: string
+                required: true
+                example: 24a8a24f-7126-4aa7-b690-259b2eaccaee
+            body:
+              application/json:
+                type: !include schemas/GetBsPeriods.json
+                examples:
+                  examples-1:
+                    description: Multiple periods returned
+                    value: !include examples/GetBsPeriods.json
+
+          400:
+            body:
+              application/json:
+                type: types.errorResponse
+                examples:
+                  missingHeader:
+                    description: One or more of the required headers are missing
+                    value:
+                      code: MISSING_HEADER
+                  invalidHeader:
+                    description: One or more of the header values are invalid
+                    value:
+                      code: INVALID_HEADER
+                  invalidNino:
+                    description: Invalid NINO format
+                    value:
+                      code: INVALID_NINO
+
+          404:
+            body:
+              application/json:
+                type: types.errorResponse
+                examples:
+                  resourceNotFound:
+                    description: The supplied NINO was not found
+                    value:
+                      code: RESOURCE_NOT_FOUND
+
+          502:
```

Comment:
Is not missing here a 500 error in case of downstream Errors ? 502 is only sent when the connection with IF cannot be established.